src/norm/private/postgres/dbtypes.nim

```diff
@@ -21,7 +21,7 @@ func dbType*(T: typedesc[int16]): string = "SMALLINT"
 
 func dbType*(T: typedesc[int32]): string = "INTEGER"
 
-func dbType*(T: typedesc[int64]): string = "BIGINT"
+func dbType*(T: typedesc[int64 | Positive | int]): string = "BIGINT"
```

Comment:
If Positive is here then Natural should be too?

Reply:
We *can* do this, but we don't have to. If we add it, it'd be to be explicit, not because the code requires it. I'd leave that up to you.

We can leave it out, as this still works:

```nim
func dbType*(T: typedesc[int64 | Positive | int ]): string = "BIGINT"
let x: Natural = 5
echo dbType(x.typeOf)
```

The reason for that is `int`, it can act as a stand-in for `Natural`, so nim counts this as "valid". 
If you want to be explicit about it, I can add it in.